docs/installation/windows.md

```diff
@@ -1,161 +1,227 @@
+```
+
+![](../img/windows-python-success.png)
+
+## Building HELICS From Source on Windows with MSYS2 ##
+### Overview ###
+This section will layout the setting up of MSYS2 to compile and install HELICS. This guide will describe all the packages and install instructions for a 64bit build.
+
+### Setting up MSYS2 ###
+MSYS2 provides a linux like terminal environment on your Windows system. MSYS2 can be installed from [here](https://www.msys2.org/). Once MSYS2 has been installed start up msys2.exe. Follow first time updates as described on the MSYS2 website. After MSYS2 has been successfully updated Some packages need to be installed in order to configure and build HELICS. The following packages need to be installed:
+- base-devel
+- mingw-w64-x86_64-toolchain
+- git
+- mingw-w64-x86_64-cmake
+- mingw-w64-x86_64-boost
+- mingw-w64-x86_64-qt5 (only if you want to be able to run cmake-gui which this guide recommends.)
+- mingw-w64-x86_64-zeromq
+
+All packages can be installed by typing the following:
+```bash
+$ pacman -S --needed base-devel mingw-w64_x86_64-toolchain mingw-w64-x86_64-cmake mingw-w64-x86_64-boost mingw-w64-x86_64-qt5
+```
+After all this has been done /mingw64/bin must be in the PATH environment variable. If it isn't then it must be added.
+```bash
+$ export PATH=$PATH:/mingw64/bin
+```
+
+### Download HELICS Source Code ###
+Now that the MSYS2 environment has been setup and all prerequisite packages have been installed the source code can be compiled and installed. The HELICS source code can be cloned from GitHub by performing the following:
+```bash
+$ git clone https://github.com/GMLC-TDC/HELICS-src.git
+```
+git will clone the source code into a folder in the current working directory called HELICS-src. This path will be refered to by this guide as HELICS_ROOT_DIR.
+
+### Compiling HELICS From Source ###
+Change directories to HELICS_ROOT_DIR. Create a directory called helics-build. This can be accomplished by using the mkdir command. cd into this directory. Now type the following:
+```bash
+$ cmake-gui ../
+```
+If this failes that is because mingw-w64-x86_64-qt5 was not installed. If you did install it the cmake gui window should pop up. click the Advanced check box next to the search bar. Then click Configure. A window will pop up asking you to specify the generator for this project. Select MSYS Makesfiles from the dropdown menu. Then click the Specify native compilers check box and click next. Enter C:/msys64/mingw64/bin/gcc.exe for the C compiler and C:/msys64/mingw64/bin/g++.exe for the C++ compiler and click finish. Once the Configure process finished several variables will show up highlighted in red. Since this is the first time setup the Boost and ZeroMQ library. Below are the following cmake variables that need to be verified.
+
+* BUILD_CXX_SHARED_LIB should be checked as GridLAB-D dynamically links with the shared c++ library of HELICS
+* Boost_CHRONO_LIBRARY_DEBUG/RELEASE C:/msys64/mingw64/bin/libboost_chrono.dll

```

Comment:
on my system all the boost libraries have the -mt extension,  It will also depend on whether you are using static libraries or dynamic ones.   So there maybe should be a note that the exact boost libraries will depend on the other options and the particular system and file location.  What matters is that they are found

Reply:
I edited the file to reflect the standard MSYS2 boost package. unless they are compiling boost themselve which isn't necessary in the MSYS2 shell as there is a boost package that is new enough to build HELICS.